### PR TITLE
Updated README.md for review

### PR DESCRIPTION
**Pull request by ppmarq1**
I have published the website, but I think I have done it by mistake ( https://ppmarq1.github.io/Mobile-Version/ ). Please let me know if I need some actions here, because I am not sure.

thank you!🙂

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Setup and Mobile Version
+## Project Title: Pedro Marques Portfolio
 
 
-The 2 sections of this skeleton project has:
+https://ppmarq1.github.io/Mobile-Version/
```
Comment (prolajumokeoni) [inline README.md]: - When I click on the link it takes me to GitHub pages and not your portfolio website
- [x] Kindly ensure you are deploying the right branch in your settings, ensure to select `/ (root)` not docs
- Here is an example of my deployed website: 
Screenshot_4


-  please mention me on this pr if you need more clarifications

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @ppmarq1 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ GitHub flow was correctly used
✔️  Good design



## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ppmarq1** [comment]
I have changed to the correct branch for deploying. The link is https://ppmarq1.github.io/Mobile-Version/

 Thank you!!😊

**Review by gemmen29 — APPROVED** [review]
Hi @ppmarq1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**